PrivacyPreservingTokenizer.ts

```diff
@@ -0,0 +1,87 @@
+
+import { createHash } from 'crypto';
+
+
+export class PrivacyPreservingTokenizer {
+  HASH_FUNCTION: string = 'sha256';
+
+  bloomFilterLength: number;
+  numberOfHashFunctions: number;
+  eta: number; // Probability of *not* flipping a bit
```

Comment:
`eta` isn't a very helpful name. Can we rename this?

Reply:
I have changed it to `flipBitProbability`, which is the *opposite* of `eta`, that is `1-eta`. But I think it is easier to understand than `preserveBitProbability`, which is what I had thought of initially.

I have updated the other functions and tests to change from `eta` to `1-eta`.